sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -629,7 +629,9 @@ public Node visitUpdate(SqlBaseParser.UpdateContext context) {
         return new Update(
             (Relation) visit(context.aliasedRelation()),
             assignments,
-            visitIfPresent(context.where(), Expression.class));
+            visitIfPresent(context.where(), Expression.class),
+            getReturningClause(context.returning())
```

Comment:
maybe `getReturningItems`, the `clause` suffix is not really informative.

Reply:
👍